src/Tasks/AssemblyDependency/GlobalAssemblyCache.cs

```diff
@@ -373,10 +373,10 @@ internal static string GetGacPath()
         {
             int gacPathLength = 0;
             NativeMethods.GetCachePath(AssemblyCacheFlags.GAC, null, ref gacPathLength);
-            StringBuilder gacPath = new StringBuilder(gacPathLength);
+            char[] gacPath = new char[gacPathLength + 1];
```

Comment:
I don't believe you need the `+1`. Please verify before/after that this is returning the same string, i.e. with the same `Length`.

Reply:
I ran the tests and debugged it to work it out.